src/procedures/procedure.c

```diff
@@ -58,7 +59,10 @@ ProcedureCtx *ProcCtxNew(const char *name,
 
 ProcedureCtx *Proc_Get(const char *proc_name) {
 	if(!__procedures) return NULL;
-	ProcGenerator gen = raxFind(__procedures, (unsigned char *)proc_name, strlen(proc_name));
+	char *proc_name_lower = rm_malloc(strlen(proc_name)+1);
+    for (int i=0; i<strlen(proc_name); i++) proc_name_lower[i] = tolower(proc_name[i]);
+	ProcGenerator gen = raxFind(__procedures, (unsigned char *)proc_name_lower, strlen(proc_name));
```

Comment:
No need to compute `strlen(proc_name)` twice, save first call to a local variable.